app/javascript/mastodon/features/emoji/emoji_mart_data_light.ts

```diff
@@ -30,9 +30,8 @@ const emojis: Emojis = {};
 // decompress
 Object.keys(shortCodesToEmojiData).forEach((shortCode) => {
   const [_filenameData, searchData] = shortCodesToEmojiData[shortCode];
-  const native = searchData[0];
+  const [native, _short_names, search, _unified] = searchData;
   let short_names = searchData[1];
-  const search = searchData[2];
   let unified = searchData[3];
```

Comment:
You could use `let [_native, short_names, _search, unified] = searchData;` here

Reply:
Fixed. Thanks.